.github/workflows/ci.yml

```diff
@@ -4,20 +4,27 @@ on: ['push', 'pull_request']
 
 jobs:
   build-and-test:
-    name: Java ${{ matrix.java }}
+    name: Gradle ${{ matrix.gradle }} on Java ${{ matrix.java }}
     runs-on: ubuntu-latest
     environment: Bintray
 
     strategy:
+      fail-fast: false
```

Comment:
Why did you add this? I guess we want tests to pass on all Gradle and Java versions so failing fast will preserve GitHub Actions credits.

Reply:
I did not think of the credits. During development, I needed feedback for all Gradle versions for the same commit to pinpoint bugs. `fail-fast: true` would have given me much less of a signal, especially when trying to fix one Gradle version might fail the other. I'll revert it: the toggle can be turned on as needed, and in the meantime `fail-fast: true` will save on resources.